### ISSUE TITLE
local variable 'which_align' referenced before assignment

### ISSUE THREAD

**Issue by azzar1** [Guake/guake#1225]
Downstream bug: https://bugs.launchpad.net/ubuntu/+source/guake/+bug/1760858

**Comment by azzar1** [comment]
Fix proposed here: https://github.com/Guake/guake/pull/1226

**Comment by logileifs** [comment]
I have Guake Terminal 3.0.5 on Ubuntu 18.04 and I'm still getting this bug. Is there a release containing the fix I can get somewhere? I am already using the latest version available with apt-get on Ubuntu

**Comment by makslevental** [comment]
same as @logileifs

**Comment by zhanglistar** [comment]
The same issue  @logileifs

**Comment by midnight-wonderer** [comment]
Another "me too" from me, but I moved on.  
Someone already packaged newer versions of guake for newer Ubuntu distributions.  
It is doubtful that anyone would package them for Bionic Beaver.

So I moved on with
* Remove the packaged version (and note down the dependencies on removal)
* Install the current version with pip  
~~~bash
pip3 install --upgrade --user guake
~~~
* And then install the missing dependencies. (If any)

Pip seems to be the canonical distribution channel too.  
So, I guess, I will keep using the version from pip from now on.
